### PR TITLE
Feature/266/node 12

### PR DESCRIPTION
**Pull request by s1hofmann**
This PR enables node 12 support in Sakuli.

It additionally introduces a new package "e2e" which contains several e2e tests to run on CI.
Current limitations are the following:

- Travis does not support Windows hosts with GUI support at the moment, so tests are only running on Linux and macOS
- On macOS there's an ongoing problem with lts/erbium (https://github.com/nodejs/help/issues/2398), which causes segfaults on macOS with any node version >12.13.0. Documentation will be addressing this issue. Additionally, Travis config contains an exclude which allows the combination (node > v12.13.0 && macOS) to fail to account for this issue

### CODE REVIEW

File: packages/e2e/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# `integration-tests`
+
+> TODO: description
+
+## Usage
+
```
Comment (cliffle) [inline packages/e2e/README.md]: Do we need this file?

File: packages/e2e/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+wetter-light/**/*
```
Comment (cliffle) [inline packages/e2e/.gitignore]: did you copy paste this? :see_no_evil:

Comment (s1hofmann) [reply packages/e2e/.gitignore]: Shameless copy / paste 🙈

File: .travis.yml [hunk 3]
```diff
@@ -1,59 +1,82 @@
...
 
+before_install:
+  - "bash ./.build/setup_${TRAVIS_OS_NAME}.sh"
 install: true
-
 script:
-  - travis_wait npm ci
-  - lerna run build > /dev/null 2>&1
-  - FIREFOX_WD_URL=http://localhost:4444/wd/hub CHROME_WD_URL=http://localhost:4445/wd/hub npm test -- --runInBand --ci --bail
-  - sonar-scanner
```
Comment (cliffle) [inline .travis.yml]: I just noticed that sonar cloud is not reporting back. Is it because this is missing in e.g. `run_linux.sh`?

**Comment by tnobody** [comment]
I really like the build matrix results which are rendered in the GH overview: https://github.com/sakuli/sakuli/pull/330/checks?check_run_id=429399360

Maybe we could utilize it to generate a Technical support table for the website?

**Pull request merged**